api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -2,23 +2,6 @@
   <service>api</service>
   <runtime>java11</runtime>
 
-  <!--
-  The GAE java8 runtime restricts certain JVM APIs. The java11 runtime has fewer restrictions.
-  Google provides GAE-specific libraries to work around these restrictions, but using these
-  libraries on the java11 runtime throws a:
-    com.google.apphosting.api.ApiProxy$FeatureNotEnabledException
-
-  See:
-    https://cloud.google.com/appengine/docs/standard/bundled-services-overview
-    https://cloud.google.com/appengine/docs/standard/java-gen2/services/access
-    https://www.googlecloudcommunity.com/gc/Serverless/FeatureNotEnabledException-using-App-Engine-bundled-service/m-p/181253
-
-  The following flag enables these libraries in the java11 runtime, allowing us to migrate away
-  from them in subsequent steps.
-  -->
-  <app-engine-apis>true</app-engine-apis>
-
-  <threadsafe>true</threadsafe>
```

Comment:
The `threadsafe` element hasn't been supported since we moved to the new runtime.